From: Philipp Thomas <pth@suse.de>
Date: 2013-01-07 12:50:41+01:00
Subject: Fix quoting of function definitions
References: bnc#794807

Newer automakes warn if the arguments to AC_DEFUN aren't quoted properly.

---
 libstroke.m4 |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: libstroke.m4
===================================================================
--- libstroke.m4.orig	2001-07-05 14:44:54.000000000 +0200
+++ libstroke.m4	2013-01-07 12:49:22.644977581 +0100
@@ -26,7 +26,7 @@ dnl libname             (optional) actua
 dnl                     if different from name
 dnl description         (optional) used to construct help string
 dnl
-AC_DEFUN(smr_ARG_WITHLIB, [
+AC_DEFUN([smr_ARG_WITHLIB], [
 
 ifelse($2, , smr_lib=[$1], smr_lib=[$2])
 
@@ -74,7 +74,7 @@ dnl header              a header file re
 dnl extra-flags         (optional) flags required when compiling the
 dnl                     header, typically more includes; for ex. X_CFLAGS
 dnl
-AC_DEFUN(smr_ARG_WITHINCLUDES, [
+AC_DEFUN([smr_ARG_WITHINCLUDES], [
 
 AC_ARG_WITH([$1]-includes,
 [  --with-$1-includes=DIR  set directory for $1 headers],
@@ -116,7 +116,7 @@ dnl header      (optional) header requir
 dnl x-libs      (optional) extra libraries, if needed to link with lib
 dnl x-flags     (optional) extra flags, if needed to include header files
 dnl
-AC_DEFUN(smr_CHECK_LIB,
+AC_DEFUN([smr_CHECK_LIB],
 [
 ifelse($2, , smr_lib=[$1], smr_lib=[$2])
 ifelse($5, , , smr_header=[$5])
